authoringAssistant/settings.json

```diff
@@ -16,11 +16,12 @@
   "base-path-library": "node_modules/@explorable-viz/fluid/dist/fluid/fluid",
   "fluid-common-folder": "./fluid-common",
   "enable-reasoning": true,
+  "split-multiple-replace-tag": true,
   "learning-case-folder": "learningCases",
-  "test-case-folder": "testCases/methane-emissions",
+  "test-case-folder": "testCases/scigen",
```

Comment:
Is this because the query loading code doesn’t support subfolders?

Reply:
Yes. If we need I can implement this

Reply:
Yes I think that would be good so that all the tests can be run together